cdap-ui/app/cdap/components/Header/ProductDropdown/index.js

```diff
@@ -148,14 +149,16 @@ export default class ProductDropdown extends Component {
             </div>
           </DropdownToggle>
           <CustomDropdownMenu right>
-            <DropdownItem
-              tag="li"
-              onClick={this.toggleAboutPage}
-            >
-              <a>{T.translate('features.Navbar.ProductDropdown.aboutLabel', {
-                productName: Theme.productName,
-              })}</a>
-            </DropdownItem>
+            <If condition={Theme.showAboutProductModal}>
```

Comment:
nit: we are being explicit in other places `Theme.showAboutProductModal === true`